### ISSUE TITLE
trying to create offline paper wallet

### ISSUE THREAD

**Issue by farawayfarm** [kvhnuke/etherwallet#518]
I downloaded the latest release (zip file), went offline and ran the index.html file in Firefox. All I got was source code, not anything that looked like the MyEtherWallet website. Any ideas on what I'm doing wrong? I've tried several times - same result.

**Comment by tayvano** [comment]
Did you move the index.html file, or the entire zip folder?

**Comment by farawayfarm** [comment]
I saved it in my downloads and opened the folder. There were several files in the folder

**Comment by farawayfarm** [comment]
11 files, including the index.html file

**Comment by farawayfarm** [comment]
I also tried running it while connected to the internet, with the same result. Just source code...

**Comment by crptm** [comment]
@farawayfarm can you do a screenshot or something?

**Comment by crptm** [comment]
@farawayfarm you downloaded dist-x.x.x.zip ?

**Comment by farawayfarm** [comment]
MyEtherWallet
Open-Source & Client-Side Ether Wallet · v3.8.4    {{curLang}}

    Deutsch
    Ελληνικά
    English
    Español
    Suomi
    Français
    Magyar
    Bahasa Indonesia
    Italiano
    日本語
    한국어
    Nederlands
    Norsk Bokmål
    Polski
    Português
    Русский
    Türkçe
    Tiếng Việt
    简体中文
    繁體中文
    Disclaimer

   {{curNode.name}} ({{curNode.service}})

    {{value.name}} ({{value.service}}) [Remove Custom Node]
    Add Custom Node

«

»
Set Up Your Custom Node

To connect to a local node...

    URL: http://127.0.0.1
    Port: 8545
    Run geth: geth --rpc --rpccorsdomain "null" --keystore "dont_put_secret_files_here_ever"
    Run Parity: parity --rpccorsdomain "*" --keys-path "dont_put_secret_files_here_ever"

Your node must be HTTPS in order to connect to it via MyEtherWallet.com. You can [download the MyEtherWallet repo & run it locally](https://github.com/kvhnuke/etherwallet/releases/latest) to connect to any node. Or, get free SSL certificate via [LetsEncrypt](https://letsencrypt.org/).
Node Name
URL
Port
HTTP Basic access authentication
User
Password
ETH ETC Ropsten Kovan Rinkeby Custom Supports EIP-155
Chain ID
Generate Wallet
Enter password

Generate Wallet

**Comment by crptm** [comment]
@farawayfarm is there any chance you have javascript disabled in firefox? what is your firefox version?

**Comment by farawayfarm** [comment]
Firefox 53.0.3 (32 bit)

**Comment by crptm** [comment]
works for me, it has to be js thing.

**Comment by farawayfarm** [comment]
Maybe it was because I had pop-up windows blocked?

**Comment by crptm** [comment]
Unlikely, but you might have some other extension, try in private browsing window?

**Comment by farawayfarm** [comment]
I tried that, and it opens in the current firefox window. I'm talking to you via Firefox, so I can't shut this window down. Still trying to figure out the Javascript on this version

**Comment by crptm** [comment]
@farawayfarm drag'n'drop file to private window

**Comment by tayvano** [comment]
FYI I have another report of this in the inbox. Can't for the life of me figure out what changed though. The guy in the inbox has tried a few different zips of different versions. :(

> When I click on it, it opens a blank page with a choice of languages and clicking on English does nothing. I’ve been trying different things now for 2 days and getting the same result.
> I can create the Ether Wallet, but I wanted to do it OFFLINE, and the zip file/Index was supposed to do this, but all I get is a page with a vertical column of language choices. Strange.
>The link you sent does the sam thing. Thanks anyway.

He has said he has tried Chrome and Safari. 

?? 😑 ??

**Comment by farawayfarm** [comment]
Well, it's comforting to know that it's not just me!

**Comment by farawayfarm** [comment]
I tried the drag and drop in a private window, but can't seem to get that to work either. 
I was so stoked... I felt like I finally had a pretty good handle on what I was doing... been reading everything I can find about making an offline wallet.  Darn, go figure the first step would be a game-stopper. 
Thanks for trying to help, and so quickly.

**Comment by crptm** [comment]
@farawayfarm can you open console https://developer.mozilla.org/en/docs/Tools/Web_Console/Opening_the_Web_Console refresh page and paste what you've got there?

**Comment by farawayfarm** [comment]
To open the Web Console:

    either select "Web Console" from the Web Developer submenu in the Firefox Menu (or Tools menu if you display the menu bar or are on Mac OS X)
    or press the CtrlShiftK (CommandOptionK on OS X) keyboard shortcut.

The Toolbox will appear at the bottom of the browser window, with the Web Console activated (it's just called "Console" in the DevTools toolbar):

The Web Console's interface is split into three horizontal sections:

    Toolbar: along the top is a toolbar containing buttons with labels like "Net", "CSS", and "JS". This toolbar is used to filter which messages are displayed
    Command Line: along the bottom is a command line that you can use to enter JavaScript expressions
    Message Display Pane: in between the toolbar and the command line, and occupying most of the window, is the space in which the Web Console displays messages

**Comment by crptm** [comment]
@farawayfarm yes, follow that instruction :) hit ctrl+shift+k in the tab you have wallet opened, then refresh page, then say or screenshot what you see :)

**Comment by farawayfarm** [comment]
Sorry, I've been trying to figure out how to paste the screen shot. 

I get a divided screen, The top half is the wallet in code as usual, the bottom half is this:

Inspector  Console  Debugger  Style Editor   Performance   Memory   Network
Net  CSS   JS  Security  Logging  Server

**Comment by crptm** [comment]
yep, when you refresh the tab (ctrl+R) with devided screen, do you get any messages in the bottom half?

**Comment by farawayfarm** [comment]
Nope, nothing

**Comment by crptm** [comment]
@farawayfarm open in any tab about:config, search for javascript.enabled, make sure that it is set to true

**Comment by farawayfarm** [comment]
javascript.enabled;true

**Comment by farawayfarm** [comment]
there are a few javascript options that are set to false. Do you want to know which ones?

**Comment by crptm** [comment]
@farawayfarm when you open the file what is the url?

**Comment by farawayfarm** [comment]
file:///C:/Users/LT/AppData/Local/Temp/Temp1_dist-v3.8.4(2).zip/index.html

**Comment by crptm** [comment]
ok, can you extract zip first and then try to run it?
